models/articles.model.js

```diff
@@ -1,7 +1,11 @@
 const db = require("../db/connection")
 
 exports.fetchArticleById = (article_id) => {
-    return db.query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+    return db.query(`SELECT articles.*, COUNT(comments.comment_id) AS comment_count 
```

Comment:
Nice work getting the count in - have a look at casting if you would like to alter the data type :)